### PR TITLE
db restructuring and updates for SIWE based access and other future features!

### PR DESCRIPTION
**Pull request by memosys**
have reset migrations; db needs to be tear down and tear up

### CODE REVIEW

File: web-portal/backend/src/decorators/roles.decorators.ts
Comment (plor) [inline web-portal/backend/src/decorators/roles.decorators.ts:1]: empty file?

File: web-portal/backend/src/guards/auth.guard.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class AuthGuard implements CanActivate {
+  canActivate(
+    context: ExecutionContext,
+  ): boolean | Promise<boolean> | Observable<boolean> {
+    console.log('Inside AuthGuard');
+    const request = context.switchToHttp().getRequest();
+    return true; // @note: temp  returning true for testing
```
Comment (plor) [inline web-portal/backend/src/guards/auth.guard.ts:11]: Is this for local testing or did you mean for this to make it to `develop`?

**Review by plor — APPROVED** [review]
Seems reasonable, just a couple questions. Also, we should avoid doing this sort of migration reset going forward. Bad habits and all that.

File: web-portal/backend/src/guards/auth.guard.ts (hunk 1 above)
Comment (memosys) [reply web-portal/backend/src/guards/auth.guard.ts:11]: Didn't mean to but didn't revert; working off this!

**Pull request merged**